cmds/wifi/wifi.go

```diff
@@ -73,6 +73,10 @@ func init() {
 }
 
 func scanWifi() error {
+	if *test {
```

Comment:
hmm. could you just have a function called doNothing, and a function called doScanWiFi, and the have this
scanWiFi = doScanWiFi
and then
if *test {
scanWiFi = doNothing
}
?

This way of doing it just seems weird to me

Reply:
Fixed